documentation/developers.md

```diff
@@ -88,10 +89,29 @@ web-router-service: {
   }
 }
 ```
-
 For information on proxying plaintext `/status` requests to an otherwise HTTPS
 protected server, see the [Status Proxy documentation](./status-proxy-service.md).
 
+The status service can interact with the tk-auth service if it is also a
+dependency. If it is a dependency, it is possible to require authentication for
```

Comment:
It'd be good to distinguish between jar/library dependencies (which tk-auth is now one of for tk-status) vs services loaded during application runtime. Maybe something like, "if it's included in the trapperkeeper application's bootstrap config" rather than "dependency".

Reply:
It's not included in the trapperkeeper application's bootstrap config, is it?  I thought it was in the application's (like puppetserver or puppetdb) bootstrap config file.

Reply:
I don't think that's a distinction I meant to make, I used "trapperkeeper application" as a generalization of puppetdb, puppetserver, etc. ...Applications using trapperkeeper.